cmd/booster-bitswap/init.go

```diff
@@ -24,10 +24,6 @@ func configureRepo(cfgDir string, createIfNotExist bool) (peer.ID, crypto.PrivKe
 		return "", nil, fmt.Errorf("%s is a required flag", FlagRepo.Name)
 	}
 
-	if err := os.MkdirAll(cfgDir, 0744); err != nil {
```

Comment:
This change fixes a problem where a directory called ~ could be created in the current working directory